app/views/content_items/briefing.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render 'govuk_publishing_components/components/title', @content_item.title_and_context %>
+  </div>
+  <%= render 'shared/translations' %>
+  <div class="govuk-grid-column-two-thirds">
+    <%= render 'govuk_publishing_components/components/lead_paragraph', text: @content_item.description %>
+  </div>
+</div>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds content-bottom-margin">
+    <div class="responsive-bottom-margin">
+      <%= render "govuk_publishing_components/components/govspeak", {
+        direction: page_text_direction,
+      } do %>
+        <%= raw(@content_item.body) %>
+      <% end %>
+
+      <%= render "govuk_publishing_components/components/govspeak", {
+        direction: page_text_direction,
+      } do %>
+        <%= raw("<a href='https://www.youtube.com/watch?v=DlmmT_jjHKw'>video</a>") %>
+      <% end %>
```

Comment:
Don't think we'll need this - the YouTube link will be in the `@content_item.body`.

Reply:
yh I put that in to show you how it would look :)